Scripts/script-ExtractDomainFromUrlFormat.yml

```
@@ -10,11 +10,19 @@ script: |-
 
   inp = demisto.args()['input']
 
+  results = []
+
   is_email = validate_email(inp)
   if is_email:
-      domain = re.search("@[\w.]+", inp)
+      domain = re.search("@[\w.|\-]+", inp)
       if len(domain.group()) > 0:
-          demisto.results(domain.group()[1:])
+          extractedDomain = domain.group()[1:]
+          results.append(extractedDomain)
+          _list = extractedDomain.split(".")
+          if len(_list) > 2:
+              parentDomain = _list[len(_list)-2] + "." + _list[len(_list)-1]
```

why specifically the last two?

```suggestion
              parentDomain = '.'.join(_list[-2:])
```

---

seems reasonable, thanks for the clarification.